src/sdk/models/smil-model.js

```diff
@@ -0,0 +1,760 @@
+//  LauncherOSX
```

Comment:
I think SmilModel is needed. The index.js creates a new instance of SmilModel which is extended in the index file but originates in the smil-model.js